ogr/services/forgejo/issue.py

```diff
@@ -37,4 +145,46 @@ def get_list(
         assignee: Optional[str] = None,
         labels: Optional[list[str]] = None,
     ) -> list["Issue"]:
-        raise NotImplementedError("TBD")
+        if not project.has_issues:
+            raise IssueTrackerDisabled()
+
+        parameters: dict[str, Union[str, list[str], bool]] = {
+            "state": status.name,
+            "type": "issues",
+        }
+        if author:
+            parameters["created_by"] = author
+        if assignee:
+            parameters["assigned_by"] = assignee
+        if labels:
+            parameters["labels"] = labels
+        try:
+            issues_paginator = paginate(
+                project.service.api.issue.list_issues,
+                owner=project.namespace,
+                repo=project.repo,
+                **parameters,
+            )
+            return [ForgejoIssue(issue, project) for issue in issues_paginator]
+        except NotFoundError as ex:
+            if "user does not exist" in str(ex):
+                return []
+            raise OperationNotSupported(f"Failed to list issues {ex}") from ex
```

Comment:
I don't think it's a good idea, we should wrap everything, so in this case, I would just let any issue get wrapped in the `ForgejoAPIException` rather than specifically handle cases like this.